src/mmw/js/src/modeling/models.js

```diff
@@ -80,7 +82,127 @@ var ResultModel = Backbone.Model.extend({
         polling: false, // True if currently polling
         active: false, // True if currently selected in Compare UI
         activeVar: null // For GWLFE, the currently selected variable in the UI
-    }
+    },
+
+    toTR55RunoffCSV: function(isCurrentConditions, aoiVolumeModel) {
+        var rows = ['"Runoff Partition","Water Depth (cm)","Water Volume (m3)"'],
+            resultKey = isCurrentConditions ? 'unmodified' : 'modified',
+            result = this.get('result')['runoff'][resultKey],
+            labels = [['runoff', 'Runoff'],
+                      ['et'    , 'Evapotranspiration'],
+                      ['inf'   , 'Infiltration']];
+
+        return rows.concat(labels.map(function(label) {
+            var key = label[0],
+                partition = label[1],
+                depth = result[key],
+                volume = aoiVolumeModel.adjust(depth),
+                row = [
+                        partition,
+                        round(depth, 3),
+                        round(volume, 2)
+                    ].join('","');
+
+            return '"' + row + '"';
+        })).join('\n');
+    },
+
+    toTR55WaterQualityCSV: function(isCurrentConditions, aoiVolumeModel) {
+        var rows = ['"Quality Measure","Load (kg)","Loading Rate (kg/ha)","Average Concentration (mg/L)"'],
+            resultKey = isCurrentConditions ? 'unmodified' : 'modified',
+            results = this.get('result')['quality'][resultKey];
+
+        return rows.concat(results.map(function(result) {
+            var load = result.load,
+                loadingRate = aoiVolumeModel.getLoadingRate(load),
+                adjustedRunoff = aoiVolumeModel.adjust(result.runoff),
+                concentration = adjustedRunoff ? load / adjustedRunoff : 0,
```

Comment:
There's a discrepancy between the downloaded and table concentration. I believe we sneakily convert this value from grams to milligrams, and need to do so here.
https://github.com/WikiWatershed/model-my-watershed/blob/2afb33243b187e9b64bef42d2d8db442a041c008/src/mmw/js/src/modeling/tr55/quality/views.js#L98

<img width="662" alt="screen shot 2018-03-05 at 5 38 27 pm" src="https://user-images.githubusercontent.com/7633670/37003787-3acf22aa-209c-11e8-98d2-498d5e9dcfff.png">
<img width="459" alt="screen shot 2018-03-05 at 5 38 23 pm" src="https://user-images.githubusercontent.com/7633670/37003788-3addcb34-209c-11e8-89ab-daa7b63930d3.png">